elements/lisk-bft/package.json

```
@@ -24,8 +24,8 @@
 	"scripts": {
 		"clean": "./scripts/clean.sh",
 		"format": "prettier --write '**/*'",
-		"lint": "tslint --format verbose --project .",
-		"lint:fix": "npm run lint -- --fix",
+		"lint": "eslint . --ext .js,.ts",
+		"lint:fix": "eslint --fix --ext .js,.ts .",
```

It's missing a `.` in command, but on other packages, we used different commands. 

```suggestion
		"lint:fix": "npm run lint -- --fix",
```

---

it suppose to have the `dot` and it looks correct?

---

Missed that dot at the end. Can we do it the way for lint command.